encoder/vaapiencoder_base.cpp

```diff
@@ -450,6 +452,19 @@ const ProfileMapItem g_profileMap[] = {
     { PROFILE_H265_MAIN10, VAProfileHEVCMain10 },
 };
 
+#else
+
+const ProfileMapItem g_profileMap[] = {
+    { PROFILE_H264_BASELINE, VAProfileH264Baseline },
+    { PROFILE_H264_CONSTRAINED_BASELINE, VAProfileH264ConstrainedBaseline },
+    { PROFILE_H264_MAIN, VAProfileH264Main },
+    { PROFILE_H264_HIGH, VAProfileH264High },
+    { PROFILE_JPEG_BASELINE, VAProfileJPEGBaseline },
+};
+
+#endif
+
```

Comment:
@uartie, agree with you. @yizhouwei you can check  VA_CHECK_VERSION defined in libva project, and you can refer https://github.com/01org/gstreamer-vaapi/blob/master/gst-libs/gst/vaapi/gstvaapiutils.c#L137